environments/staging/staging-platform-hmcts-net.yml

```diff
@@ -6,8 +6,11 @@ vnet_links:
     vnet_id: "/subscriptions/ed302caf-ec27-4c64-a05e-85731c3ce90e/resourceGroups/reformMgmtCoreRG/providers/Microsoft.Network/virtualNetworks/reformMgmtCoreVNet"
   - link_name: "bastion-nonprod-vnet"
     vnet_id: "/subscriptions/b44eb479-9ae2-42e7-9c63-f3c599719b6f/resourceGroups/bastion-nonprod-rg/providers/Microsoft.Network/virtualNetworks/bastion-nonprod-vnet"
-A: []
-
+A:
+  - name: hmi-pact
```

Comment:
i think we've had this discussion before is this a pact broker? and if so why aren't you using the platform managed one at https://pact-broker.platform.hmcts.net ?